app/views/audits/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<% add_parent_breadcrumb_link "Home", planning_applications_path %>
+<% add_parent_breadcrumb_link "Application", planning_application_path(@planning_application) %>
+
+<% content_for :title, "Audit log" %>
+
+<h1 class="govuk-heading-l" style="margin-bottom: 7px;">
+  <%= @planning_application.reference %>
+</h1>
+<p class="govuk-body">
+  <%= @planning_application.site.full_address %>
+</p>
+
+<table class="govuk-table">
+  <caption class="govuk-table__caption govuk-table__caption--m">Activity Log</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Date</th>
+      <th scope="col" class="govuk-table__header">User</th>
+      <th scope="col" class="govuk-table__header">Activity</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <% @audits.each do |audit_item| %>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell"><%= audit_item.created_at.strftime('%d-%m-%Y %H:%M') %></td>
+        <td class="govuk-table__cell"><%= User.find_by_id(audit_item.user_id).name %></td>
+        <td colspan="2" class="govuk-table__cell">
+          <%= activity(audit_item.activity_type, audit_item.activity_information) %> <br>
+          <% if !audit_item.audit_comment.nil? %>
```

Comment:
using `if audit_item.audit_comment.present?` would be better here: more readable, plus returns false if it's an empty string.